pinot-query-runtime/src/test/java/org/apache/pinot/query/service/QueryServerTest.java

```diff
@@ -73,17 +83,59 @@ public void tearDown() {
     }
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
I think we should write a separate test class only to test the SerDe correctness of plan and StageNode